code/game/objects/structures/ladders.dm

```diff
@@ -177,7 +177,7 @@
 
 /obj/structure/ladder/unbreakable/dive_point/buoy
 	name = "diving point buoy"
-	desc = "A buoy marking the location of an underwater dive area."
+	desc = "A buoy marking the location of an underwater dive area. <span class='notice'>You can pull people and objects along with you</span>" //the notice is because several people didn't know you could do this and I feel it's important people knew
```

Comment:
You'll want `boldnotice` here because `regular` notice is not bolded.
```suggestion
	desc = "A buoy marking the location of an underwater dive area. <span class='boldnotice'>You can pull people and objects along with you</span>" //the notice is because several people didn't know you could do this and I feel it's important people knew
```